gym_lowcostrobot/envs/lift_cube_env.py

```diff
@@ -221,6 +246,8 @@ def reset(self, seed=None, options=None):
         cube_rot = np.array([1.0, 0.0, 0.0, 0.0])
         robot_qpos = np.array([0.0, 0.0, 0.0, 0.0, 0.0, 0.0])
         self.data.qpos[:] = np.concatenate([cube_pos, cube_rot, robot_qpos])
+        self.done = False
```

Comment:
For lift you don't want to terminate the episode early, otherwise the robot will just find a way to throw the cube. By not finishing the early episode, it forces the agent to learn how to stabilise the cube.

Reply:
That is a good catch, will modify it